java/arcs/core/data/testdata/Manifest2ProtoTest.textproto

```diff
@@ -0,0 +1,303 @@
+# This files is a part of 3 files suite of testdata:
```

Comment:
It seems like all of these test files should live under javatests/

Reply:
Ah... in a follow up PR someday!